proposals/all_proposals.md

```diff
@@ -0,0 +1,658 @@
+Therefore, we must establish rules on how the proceeds of technological process benefit all different parts of society. That is a quintessential democratic process: the clash of different interest groups must be done openly, through debate, with enforceable rules of engagement.
+DiEM25 strongly believes that the decision on allocation of the proceeds of technological development must be openly and democratically discussed. DiEM25 firmly rejects the dogma of the invisible hand.
+
+7. We stand on the shoulders of giants
+
+What does that mean?
+
+Technology does not fall from the sky. For tens of thousands of years, humans have made incremental progress in developing technology. It is the result of collaboration and co-operation between many. The knowledge handed down from our ancestors is absolutely necessary for us to build on it. And just like we borrowed that knowledge from our ancestors, we need to pass it on to the next generations.
+DiEM25 rejects artificial boundaries around knowledge, and wants to ensure that continued progress remains possible through the sharing of both old and new knowledge.
+
+8. No Frankenstein - principle
+
+What does that mean?
+
+Our society is becoming ever more complex, as is our technology. 50 years ago, a well-trained engineer could understand, and repair, a lot of technology. That is no longer the case. With hyper-specialization comes hyper-mutual dependency. Sometimes we don’t understand fully the technology we create. Therefore, the myth of the sole genius solving a fundamental problem in his (never, by the way, “her”) basement is no longer useful – quite the opposite.
+The ever increased specialization and complexity of technology makes it necessary for our society to open up as much information as possible about how things work – so we are able to understand what goes wrong when something goes wrong. As it inevitably will.
+DiEM25 rejects the Frankenstein myth as a workable basis for developing and maintaining knowledge and innovation. An ever more complex and specialized society and technology demands as much open knowledge and communication as possible.
+
+9. Technology reflects our values
+
+What does that mean?
+
+Technology is never value-free. The way we fund, adopt, use and regulate technology, or not, reflects society’s choice of its values and priorities. E.g. we currently accept that thousands of children are killed every year through society’s incoherent approach to the use of transportation technology. That is a reflection of our society’s priorities and values. We must be more aware of how choices around technology must be rooted in values, and openly discuss and decide on them in a democratic way. The agenda setting of the debate around technology and values should be open, and not set by the technology industry itself. DiEM25 strongly support
```

Comment:
very interesting / possible tension with earlier proposal? 

I think scenario here is quite important - company collection of data from: 

- their own workers 
- their independent providers / contractors 
- workers for another company, or some other captive / compelled population 
- consumers of their products 

these are all currently different types of data providers, and while it may not be necessary at this point, I think elucidating the difference between these scenarios and the data collected in each one might help address some conflicting perspectives (i.e. remunerate people for their data, but move away from service-for-data business model) - kate